scripts/bump-version.sh

```
@@ -0,0 +1,197 @@
+
+command -v jq >/dev/null 2>&1 || die "jq is required (brew install jq / apt install jq)"
+
+echo "Bumping version to $VERSION ..."
+
+# ── resolve repo root ─────────────────────────────────────────────────────────
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)"
+REPO_ROOT="$(cd "$SCRIPT_DIR/.." && pwd -P)"
+
+CARGO_TOML="$REPO_ROOT/Cargo.toml"
+CARGO_LOCK="$REPO_ROOT/Cargo.lock"
```

Shellcheck (0.11.0)

[warning] CARGO_LOCK appears unused. Verify use (or export if used externally).

(SC2034)